opus_codec_registration.src.html

```diff
@@ -197,6 +213,67 @@
   </dd>
 </dl>
 
+OpusSignal {#opus-signal}
+------------------------------------------
+<pre class='idl'>
+<xmp>
+enum OpusSignal {
+  "auto",
+  "music",
+  "voice",
+};
+</xmp>
+</pre>
+
+The {{OpusSignal}} describes the type of audio signal being encoded. See
+[secion 9.3.1.1](https://draft.ortc.org/#opus-codec-options*) of [[OBJECT-RTC]].
```

Comment:
Again, I'm not sure if referencing that section is allowed and adds much. The `signal` and `application` parameters are not defined with a lot of details there.

Reply:
I would keep the reference non-normative. Or you can just copy the text.